src/download_images.py

```diff
@@ -127,6 +23,7 @@ def main(
         Path,
         Argument(help="Path to Input Points File"),
     ],
+    image_source: Annotated[str, Argument(help="Where to Get Images From")],
```

Comment:
You can use an enum here for better validation and for the help text to print out the available choices: https://typer.tiangolo.com/tutorial/parameter-types/enum/

Reply:
Enum is used in latest revision